dev/vagrant/bootstrap/vault/post/01-create_vault_policy_to_read_secrets.yml

```diff
@@ -0,0 +1,14 @@
+- name: Create policy in vault to read secrets
+  shell: |
+    vault policy write kv-secret - << EOF
+    path "secret/data/random-string/*" {
+      capabilities = ["read"]
+    }
+    EOF
+  register: cmd_op
+  environment:
+    VAULT_ADDR: "{{ lookup('env', 'VAULT_ADDR') }}"
```

Comment:
Do you actually need this one? As long as the `VAULT_ADDR` in env you should be good

Reply:
Yes, I agree. Is it going to be a problem in the future due, that `VAULT_ADDR` setup implicitly here?

Reply:
It's a tradeoff between readability and less code I guess. As long as you there is a `VAULT_ADDR` env variable everything is fine. What about adding a small comment mentioning that this code implicitly uses the `VAULT_ADDR` env variable?

Reply:
Comment line vs Explicit setup?

Reply:
Honestly? I'd like a comment line just because the explicit setup is just an abomination to DRY. "Take the value of the environment variable VAULT_ADDR and create a new environment variable VAULT_ADDR with the same value"

Reply:
I would suggest just remove VAULT_ADDR line without any comment

Reply:
Fair enough. I am down with that